js/__internal/grids/grid_core/editing/m_editing_cell_based.ts

```diff
@@ -120,12 +120,16 @@ const editingControllerExtender = (Base: ModuleType<EditingController>) => class
         const rowsView = this.getView('rowsView');
         const $targetCell = $targetElement.closest(`.${ROW_CLASS}> td`);
         const rowIndex = rowsView.getRowIndex($targetCell.parent());
-        const columnIndex = rowsView.getCellElements(rowIndex).index($targetCell);
-        const visibleColumns = this._columnsController.getVisibleColumns();
-        // TODO jsdmitry: Move this code to _rowClick method of rowsView
-        const allowEditing = visibleColumns[columnIndex] && visibleColumns[columnIndex].allowEditing;
+        const cellElements = rowsView.getCellElements(rowIndex);
 
-        result = result && !allowEditing && !this.isEditCell(rowIndex, columnIndex);
+        if (cellElements?.length) {
+          const columnIndex = cellElements.index($targetCell);
+          const visibleColumns = this._columnsController.getVisibleColumns();
+          // TODO jsdmitry: Move this code to _rowClick method of rowsView
+          const allowEditing = visibleColumns[columnIndex]?.allowEditing;
+
+          result = result && !allowEditing && !this.isEditCell(rowIndex, columnIndex);
```

Comment:
Minor comment

Let's move ```this.isEditCell(rowIndex, columnIndex)``` to separate variable:
```suggestion
          const isEditableCell = this.isEditCell(rowIndex, columnIndex);

          result = result && !allowEditing && !isEditableCell;
```